src/server/scripts/EasternKingdoms/zone_dun_morogh.cpp

```diff
@@ -83,7 +87,217 @@ class npc_frozen_mountaineer : public CreatureScript
     }
 };
 
+enum DecontaminationData
+{
+    NPC_DECONTAMINATION_BUNNY           = 46165,
+    NPC_CLEAN_CANNON_X2                 = 46208,
+    NPC_SANITRON_5000                   = 46185,
+    NPC_SAFE_TECHNICIAN                 = 46230,
+
+    SAY_SANITRON_START                  = 0,
+    SAY_SANITRON_FINISH                 = 1,
+    SAY_SANITRON_DESTROY                = 2,
+
+    SAY_TECHNICIAN_CLEAN_CANNON         = 0,
+    SAY_TECHNICIAN_SANITRON_DESTROY     = 1,
+
+    PATH_SANITRON_LEFT_START            = 4618500,
+    PATH_SANITRON_MIDDLE_START          = 4618501,
+    PATH_SANITRON_RIGHT_START           = 4618502,
+
+    WAYPOINT_SANITRON_STAGE_ONE         = 0,
+    WAYPOINT_SANITRON_STAGE_TWO         = 1,
+    WAYPOINT_SANITRON_STAGE_THREE       = 2,
+    WAYPOINT_SANITRON_FINISH            = 3,
+
+    ACTION_TECHNICIAN_START_EVENT       = 1,
+
+    POINT_SAFE_TECHNICIAN_SANITRON      = 1,
+    POINT_SAFE_TECHNICIAN_FINISH        = 2,
+
+    SPELL_DECONTAMINATE_STAGE_1         = 86075,
+    SPELL_DECONTAMINATE_STAGE_2         = 86098,
+    SPELL_DECONTAMINATE_STAGE_3         = 86086,
+    SPELL_CLEAN_CANNON_CLEAN_BURST      = 86080,
+    SPELL_FORCE_CAST_GNOMEREGAN_AURA    = 86298,
+    SPELL_SANITRON_COSMETIC_EXPLOSION   = 46419
+};
+
+// 46185 - Sanitron 5000
+struct npc_sanitron_5000 : public ScriptedAI
```

Comment:
At this point i'd rather recommend using NullCreatureAI because it's rly just a scripted braindead npc that shouldn't use any of the combat mechanics of ScriptedAI